### PR TITLE
Updating the branding to 2.0.1-servicing

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

**Comment by wli3** [comment]
@MattGertz this is a branding change

**Comment by wli3** [comment]
@dsplaisted  https://github.com/dotnet/cli/commit/33b91ad316151d325c2f0042a3a6779dbc5118ee reference commit

File: build/Version.props [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <VersionMajor>2</VersionMajor>
     <VersionMinor>0</VersionMinor>
-    <VersionPatch>0</VersionPatch>
-    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">preview3</ReleaseSuffix>
+    <VersionPatch>1</VersionPatch>
+    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">servicing</ReleaseSuffix>
```
Comment (livarcocc) [inline build/Version.props]: I don't believe we have used this branding before. Why not preview1? Anyway, is this what the runtime (core-setup) is doing? We should align our branding with theirs.

@Petermarcu

Comment (wli3) [inline build/Version.props]: @Petermarcu @leecow Update to servicing are from Nick's email. Is this what we decided?

Comment (leecow) [inline build/Version.props]: Yes, this is the branding which will be used in corefx, coreclr and core-setup. @weshaggard will be submitting those PRs in the next day or so.

**Pull request merged**